src/common/CryptoUtils.ts

```diff
@@ -141,4 +220,16 @@ export class CryptoUtils {
     public static generateIdentityToken(byteSize: number = 32): string {
         return randomBytes(byteSize).toString('base64');
     }
+
+    private static getCertFromDER(derCertificate: Buffer): forge.pki.Certificate {
```

Comment:
Nit: we should probably be consistent in naming convention with methods like [`derToPem()`](https://github.com/forcedotcom/lwc-dev-mobile-core/pull/133/files#diff-0712445dd2528d1fbb3eea9100be49d400ff2d2319caf17024b6f0414b4adba9R154), or vice versa. So either `getCertFromDer()` here, or `derToPEM()` there.